src/VisualStudio/Core/Def/Implementation/Venus/ContainedDocument.cs

```diff
@@ -38,32 +31,9 @@ namespace Microsoft.VisualStudio.LanguageServices.Implementation.Venus
     internal sealed partial class ContainedDocument : ForegroundThreadAffinitizedObject, IVisualStudioHostDocument
 #pragma warning restore CS0618 // Type or member is obsolete
     {
-        private const string ReturnReplacementString = @"{|r|}";
-        private const string NewLineReplacementString = @"{|n|}";
-
-        private const string HTML = nameof(HTML);
-        private const string HTMLX = nameof(HTMLX);
-        private const string Razor = nameof(Razor);
-        private const string XOML = nameof(XOML);
-
-        private const char RazorExplicit = '@';
-
-        private const string CSharpRazorBlock = "{";
-        private const string VBRazorBlock = "code";
-
-        private const string HelperRazor = "helper";
-        private const string FunctionsRazor = "functions";
-        private const string CodeRazor = "code";
-
-        private static readonly EditOptions s_venusEditOptions = new EditOptions(new StringDifferenceOptions
-        {
-            DifferenceType = StringDifferenceTypes.Character,
-            IgnoreTrimWhiteSpace = false
-        });
-
         private static readonly ConcurrentDictionary<DocumentId, ContainedDocument> s_containedDocuments = new ConcurrentDictionary<DocumentId, ContainedDocument>();
 
-        public static ContainedDocument TryGetContainedDocument(DocumentId id)
+        public static ContainedDocument? TryGetContainedDocument(DocumentId id)
```

Comment:
```suggestion
        public static ContainedDocument? TryGetContainedDocument(DocumentId? id)
```

Since the first line of this is checking for null on the ID side of things.

Reply:
(and yes I hate this pattern but for now let's at least document what it is)